src/lfc_scripts.py

```diff
@@ -1,13 +1,61 @@
 import mwclient
 import config
+from geopy.geocoders import Nominatim
+import folium
 
-site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
-site.login(config.username, config.api_key)
+def main():
+    site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
+    site.login(config.username, config.api_key)
 
-response = site.api(
-    'torquedataconnect',
-    format='json',
-    path='/competitions/LoneStar2020/proposals'
-)
+    response = site.api(
+        'torquedataconnect',
+        format='json',
+        path='/competitions/LoneStar2020/proposals'
```

Comment:
One immediate way you can make this code more flexible is to have the competition name be a configured value (ideally a parameterized one / passed via the CLI as a `--competition {VALUE}` flag)

^ This is outside of the scope of this PR, just wanted to mention it for future consideration.

In future I would suggest having hard coded values like `LoneStar2020` be defined constants just to make it explicit that the string relies on a hard coded value and also to expose the pattern you're using to create the path.

Right now a developer inspecting this code has no way to know for sure that the path pattern is `/competitions/{COMPETITION_NAME}/proposals` -- e.g. as currently written it's possible `competitions` or `proposals` is also a dynamic attribute.

Reply:
Modified in this PR to include competition name as a constant for readability, will follow your suggestions in future changes.